File: abc/abc177/abc177_b.fs
// https://atcoder.jp/contests/abc177/submissions/16373675

let S = stdin.ReadLine()
let T = stdin.ReadLine()

let mutable result = 0

for i in [ 0..S.Length - T.Length ] do
    let mutable count = 0
    for j in [ 0..T.Length - 1 ] do
        if S.[i + j] = T.[j] then count <- count + 1
    if count > result then result <- count

T.Length - result |> stdout.WriteLine

File: abc/abc177/abc177_b.cs
// https://atcoder.jp/contests/abc177/submissions/16373897

using System;

class Program
{
    static void Main()
    {
        var S = Console.ReadLine();
        var T = Console.ReadLine();

        var result = 0;
        for (int i = 0; i <= S.Length - T.Length; i++)
        {
            var count = 0;
            for (int j = 0; j < T.Length; j++)
            {
                if (S[i + j] == T[j])
                {
                    count++;
                }
            }
            if (count > result)
            {
                result = count;
            }
        }
        Console.WriteLine(T.Length - result);
    }
}
